charts/stable/autobrr/templates/_configmap.tpl

```diff
@@ -0,0 +1,83 @@
+{{/* Define the configmap */}}
+{{- define "autobrr.configmaps" -}}
+{{- $fullname := (include "tc.v1.common.lib.chart.names.fullname" $) -}}
+
+{{- $mainPort := .Values.service.main.ports.main.port -}}
+{{- $logLevel := .Values.autobrr.log_level -}}
+
+{{- $sessionSecret := randAlphaNum 32 -}}
+
+ {{- with lookup "v1" "Secret" .Release.Namespace $fullname -}}
+   {{- $sessionSecret = index .data "sessionSecret" | b64dec -}}
+ {{- end }}
```

Comment:
there is no such secret created...

Reply:
how would i go about doing this....

Reply:
https://github.com/truecharts/charts/blob/a19798f98e7e33df3cad4a538853f8175a4fb6c7/charts/stable/wordpress/templates/_env.tpl#L55

check also common.yaml

Reply:
this would be wrong too right? https://github.com/truecharts/charts/blob/master/charts/stable/nitter/templates/_configmap.tpl